core/src/server.cpp

```diff
@@ -304,6 +314,15 @@ void Server::broadcast(AOPacket packet, AOPacket other_packet, TARGET_TYPE targe
     }
 }
 
+void Server::unicast(AOPacket f_packet, int f_client_id)
+{
+    AOClient* l_client = getClientByID(f_client_id);
+    if (!(l_client == nullptr)) { // This should never happen, but safety first.
```

Comment:
```suggestion
    if (l_client != nullptr) { // This should never happen, but safety first.
```
why did you do it like this

Reply:
actually no wait you could even just do 
```suggestion
    if (!l_client) { // This should never happen, but safety first.
```

Reply:
Second variation results in a clazy. I'll add your first suggestion one for clarity and a clazy-free life.